### ISSUE TITLE
Always stcks at 16% Xlabs Ksampler

### ISSUE THREAD

**Issue by Torcelllo** [XLabs-AI/x-flux-comfyui#77]
got prompt
Is model already patched? False
X:\ComfyUI\comfy\ldm\modules\attention.py:407: UserWarning: 1Torch was not compiled with flash attention. (Triggered internally at C:\actions-runner\_work\pytorch\pytorch\builder\windows\pytorch\aten\src\ATen\native\transformers\cuda\sdp_utils.cpp:555.)
  out = torch.nn.functional.scaled_dot_product_attention(q, k, v, attn_mask=mask, dropout_p=0.0, is_causal=False)
We are patching diffusion model, be patient please
Patched succesfully!
clip missing: ['text_projection.weight']
Requested to load FluxClipModel_
Loading 1 new model
loaded partially 4936.91484375 4924.23095703125 0
Unloading models for lowram load.
0 models unloaded.
Requested to load Flux
Loading 1 new model
loaded partially 4936.91484375 5222.34375 0
Sampling:   0%|                                                                                 | 0/50 [00:00<?, ?it/s]

... stops, sticks here after 16% processed
![Xlabs Sticks](https://github.com/user-attachments/assets/3450883c-3805-4c87-ab59-2894df034d4e)
  Win 10, 8Gb VRAM, 64Gb RAM

**Comment by GPU-server** [comment]
What was the solution?

**Comment by Torcelllo** [comment]
To get more VRAM  :)

**Comment by GPU-server** [comment]
> To get more VRAM :)

I have 24 as for myself
